ironic-deployment/tls/default/tls.yaml

```diff
@@ -28,6 +31,9 @@ spec:
         - name: cert-ironic-inspector-ca
           mountPath: "/certs/ca/ironic-inspector"
           readOnly: true
+        - name: cert-mariadb-ca
+          mountPath: "/certs/ca/mariadb"
+          readOnly: true
       - name: ironic-inspector
         volumeMounts:
```

Comment:
ironic-inspector also uses the database.
We'll need a PR to add support for TLS in ironic-inspector-image before this can be merged also.

Reply:
inspector uses sqlite afaik, so no changes there for now I would think

Reply:
Oh, you're right. Carry on then.